selfdrive/boardd/boardd.cc

```diff
@@ -48,22 +54,37 @@ bool usb_connect() {
   err = libusb_claim_interface(dev_handle, 0);
   if (err != 0) { return false; }
 
+  #ifdef DEBUG_WITH_PANDA_LOOPBACK
+  libusb_control_transfer(dev_handle, 0xc0, 0xe5, 1, 0, NULL, 0, TIMEOUT);
+  #endif
+
   // power off ESP
   libusb_control_transfer(dev_handle, 0xc0, 0xd9, 0, 0, NULL, 0, TIMEOUT);
 
   // forward CAN1 to CAN3...soon
   //libusb_control_transfer(dev_handle, 0xc0, 0xdd, 1, 2, NULL, 0, TIMEOUT);
-  
+
   // set UART modes for Honda Accord
   for (int uart = 2; uart <= 3; uart++) {
     // 9600 baud
-    libusb_control_transfer(dev_handle, 0xc0, 0xe1, uart, 9600, NULL, 0, TIMEOUT);
+    libusb_control_transfer(dev_handle, 0x40, 0xe1, uart, 9600, NULL, 0, TIMEOUT);
     // even parity
-    libusb_control_transfer(dev_handle, 0xc0, 0xe2, uart, 1, NULL, 0, TIMEOUT);
+    libusb_control_transfer(dev_handle, 0x40, 0xe2, uart, 1, NULL, 0, TIMEOUT);
     // callback 1
-    libusb_control_transfer(dev_handle, 0xc0, 0xe3, uart, 1, NULL, 0, TIMEOUT);
+    libusb_control_transfer(dev_handle, 0x40, 0xe3, uart, 1, NULL, 0, TIMEOUT);
   }
 
+  // TODO: Boardd should be able to set the baud rate
+  // May need to set endianess.
```

Comment:
All reasonable phones are little endian. Some ARM are big endian, and some really crazy ARM have a runtime settable flag! But all modern phones are little.